### PR TITLE
WIP feature/letkf

### PR DESCRIPTION
**Pull request by flampouris**
- This PR cleans the scripts for the H(x)  for the LETKF

- WIP

### CODE REVIEW

File: jobs/JJOB_DA_PREP [hunk 1]
```diff
@@ -145,7 +144,7 @@ ${ROOT_GODAS_DIR}/scripts/prep_3denvar_yaml.sh    \
 
 
 #----------------------------------
-if [ $case = 'en' ]; then 
+if [ $DA_METHOD = 'en' ]; then
```
Comment (guillaumevernieres) [inline jobs/JJOB_DA_PREP]: "en" is too vague, how about letkf?

File: workflow/defaults/places.yaml [hunk 2]
```diff
@@ -14,7 +14,8 @@ default_places: &default_places
   DA_SLOT_LEN: 24
 # To be removed as soon as there is an ensemble yaml file
 # Number of ensemble members
-  NO_ENS_MBR: 5
+  DA_METHOD: en #options 3dvar or en
```
Comment (guillaumevernieres) [inline workflow/defaults/places.yaml]: Same comment as above "letkf" instead of "en"

File: workflow/layout/letkf_only.yaml [hunk 3]
```diff
@@ -55,13 +29,33 @@ suite: !Cycle
...
+#          config: [ base ]
+#          J_JOB: JJOB_PREP_LETKF
+#          ENSGRP: !expand "{dimval.groupid:02d}"
+#          rocoto_command: !expand >-
+#            {rocoto_load_modules} ;
+#            /usr/bin/env ENSGRP={ENSGRP} &ROOT_GODAS_DIR;/jobs/{J_JOB}
+#          ecflow_command: !expand |
+#            export ENSGRP={ENSGRP}
+#            $ROOT_GODAS_DIR/jobs/{J_JOB}
+
```
Comment (guillaumevernieres) [inline workflow/layout/letkf_only.yaml:58]: Do you need to keep these comments?

Comment (guillaumevernieres) [reply workflow/layout/letkf_only.yaml:58]: Bug slipped through from the last PR: 
Line 64:      J_JOB: JJOB_DA_LEKTF 
should be 
                  J_JOB: JJOB_DA_**LETKF**

**Review by guillaumevernieres — CHANGES_REQUESTED** [review]
bug/typo in da_letkf_run

**Comment by flampouris** [comment]
The commented out code for the group is the next step to support the parallel run of the ensemble members.

**Comment by guillaumevernieres** [comment]
> The commented out code for the group is the next step to support the parallel run of the ensemble members.

OK, sounds good. letkf test is OK, waiting for the 2 vars to finish.

**Review by guillaumevernieres — APPROVED** [review]
:+1:

**Comment by flampouris** [comment]
one step closer :)

**Pull request merged**